src/interfaces/IArrangerConduit.sol

```diff
@@ -0,0 +1,284 @@
+     *  @return totalDeposits_ The total deposits held in the asset.
+     */
+    function totalDeposits(address asset) external view returns (uint256 totalDeposits_);
+
+    /**
+     *  @dev    Returns the total requested funds for a given asset.
+     *  @param  asset          The address of the asset.
+     *  @return totalRequestedFunds_ The total requested funds held in the asset.
+     */
+    function totalRequestedFunds(address asset)
+        external view returns (uint256 totalRequestedFunds_);
+
+    /**
+     *  @dev    Returns the total amount that can be withdrawn for a given asset.
+     *  @param  asset              The address of the asset.
+     *  @return totalWithdrawableFunds_ The total amount that can be withdrawn from the asset.
+     */
+    function totalWithdrawableFunds(address asset)
+        external view returns (uint256 totalWithdrawableFunds_);
+
+    /**
+     *  @dev    Returns the total amount of cumulative withdrawals for a given asset.
+     *  @param  asset             The address of the asset.
+     *  @return totalWithdrawals_ The total amount that can be withdrawn from the asset.
+     */
+    function totalWithdrawals(address asset)
+        external view returns (uint256 totalWithdrawals_);
+
+    /**
+     *  @dev    Returns the aggregate deposits for a given ilk and asset.
+     *  @param  ilk        The unique identifier for a particular ilk.
+     *  @param  asset      The address of the asset.
+     *  @return deposits_ The deposits for the given ilk and asset.
+     */
+    function deposits(bytes32 ilk, address asset) external view returns (uint256 deposits_);
+
+    /**
+     *  @dev    Returns the aggregate requested funds for a given ilk and asset.
+     *  @param  ilk             The unique identifier for a particular ilk.
+     *  @param  asset           The address of the asset.
+     *  @return requestedFunds_ The requested funds for the given ilk and asset.
+     */
+    function requestedFunds(bytes32 ilk, address asset)
+        external view returns (uint256 requestedFunds_);
+
+    /**
+     *  @dev    Returns the aggregate withdrawable funds for a given ilk and asset.
+     *  @param  ilk           The unique identifier for a particular ilk.
+     *  @param  asset         The address of the asset.
+     *  @return withdrawableFunds_ The withdrawableFunds funds for the given ilk and asset.
+     */
+    function withdrawableFunds(bytes32 ilk, address asset)
+        external view returns (uint256 withdrawableFunds_);
+
+    /**
+     *  @dev    Returns the aggregate cumulative withdraws for a given ilk and asset.
+     *  @param  ilk          The unique identifier for a particular ilk.
+     *  @param  asset        The address of the asset.
+     *  @return withdrawals_ The withdrawals funds for the given ilk and asset.
+     */
+    function withdrawals(bytes32 ilk, address asset) external view returns (uint256 withdrawals_);
+
```

Comment:
Updated

Reply:
Updated

Reply:
I think it would be better to return a struct here. Not a big fan of these large tuples. If the automated getter generator won't do this you will have to make a custom getter and set `fundRequests` variable to `private`.